source/drivers/MicroBitCompassCalibrator.cpp

```diff
@@ -226,8 +226,8 @@ CompassCalibration MicroBitCompassCalibrator::spherify(Sample3D centre, Sample3D
 Sample3D MicroBitCompassCalibrator::approximateCentre(Sample3D *data, int samples)
 {
     Sample3D c,t;
-    Sample3D centre = { 0,0,0 };
-    Sample3D best = { 0,0,0 };
+    Sample3D centre;
+    Sample3D best;
```

Comment:
The default constructor initialises the values to zero, but is there any specific reason to remove the `{0,0,0}`?
Also, to reduce the diff as much as possible, can we undo all the whitespace character changes?

Reply:
Same as the other initializer removals. I will remove the whitespace.